app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -148,6 +152,7 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         )
     }
 
+    lateinit var factory: LayoutInflater.Factory2
```

Comment:
nit: can we group this variable together with the inflater? They're highly related and grouping them could give cues to future developers.